tutorials/finance/07_asian_barrier_spread_pricing.ipynb

```diff
@@ -57,11 +57,32 @@
     "import numpy as np\n",
     "\n",
     "from qiskit import QuantumRegister, QuantumCircuit, Aer, execute\n",
+    "from qiskit.circuit.library import IntegerComparator\n",
     "from qiskit.aqua.algorithms import IterativeAmplitudeEstimation\n",
-    "from qiskit.aqua.circuits import WeightedSumOperator, FixedValueComparator as Comparator\n",
+    "from qiskit.aqua.circuits import WeightedSumOperator\n",
     "from qiskit.aqua.components.uncertainty_problems import UnivariatePiecewiseLinearObjective as PwlObjective\n",
     "from qiskit.aqua.components.uncertainty_problems import MultivariateProblem\n",
-    "from qiskit.aqua.components.uncertainty_models import MultivariateLogNormalDistribution"
+    "from qiskit.aqua.components.uncertainty_models import MultivariateLogNormalDistribution\n",
+    "\n",
+    "\n",
+    "class Comparator(IntegerComparator):\n",
```

Comment:
The only thing here (and the other tutorials with this class) I think is that we should have a comment on what the purpose of this class is. It's not exactly clear when reading this tutorial why we need to define this class. We're instructing people that they need to write this thing (at least for now) so it would be good to just mention why.

Reply:
comments added.